content/4_Gold/LIS.mdx

```diff
@@ -587,6 +587,41 @@ class PCB {
 ```
 
 </JavaSection>
+<PySection>
+
+```py
+import sys
+
+input = sys.stdin.read
+
+
+def find_lis_length():
```

Comment:
my guy you just changed the name :sob:

Reply:
wdym?

Reply:
execute the code directly
don't put it in some function then call the function
that's so roundabout